AK/Format.cpp

```diff
@@ -149,43 +166,133 @@ void vformat(const LogStream& stream, StringView fmtstr, Span<const TypeErasedPa
     stream << builder.to_string();
 }
 
-bool Formatter<StringView>::parse(StringView flags)
+void StandardFormatter::parse(StringView specifier)
 {
-    return flags.is_empty();
+    GenericLexer lexer { specifier };
+
+    if (StringView { "<^>" }.contains(lexer.peek(1))) {
```

Comment:
```suggestion
    if (lexer.next_is(is_any_of("<^>"))) {
```
You can find ctype-like functions compatible with the GenericLexer under its definition in https://github.com/SerenityOS/serenity/blob/e8ef2b18580af33a5eb453a354310692da5371ff/AK/GenericLexer.h#L85

Reply:
Woops, just noticed you wanted to peek the 2nd character here